shopfloor/services/location_content_transfer.py

```diff
@@ -817,6 +818,9 @@ def stock_out_package(self, location_id, package_level_id):
             # split the move to process only the lines related to the package.
             package_move.split_other_move_lines(package_move_lines)
             lot = package_move.move_line_ids.lot_id
+            # We need to set qty_done at 0 because otherwise
+            # the move_line will not be deleted
+            package_move.move_line_ids.write({"qty_done": 0})
```

Comment:
Ok, now Odoo 14.0 doesn't unreserve lines having a `qty_done > 0`: https://github.com/odoo/odoo/blob/8e11a0b9683c9593f50ab0c290a1bdf857e5141e/addons/stock/models/stock_move.py#L704-L713
I'm wondering if this is what we want to do here, if the `qty_done` it means an operator processed it, resetting it to 0 is probably not what we want :thinking:

Reply:
So how you want to achieve it ?

Reply:
I don't know yet, previously with the 13.0 implementation we didn't care about it, so this should not block the merge for now.